third_party/2.7/sqlalchemy/engine/__init__.pyi

```diff
@@ -2,5 +2,7 @@
 #
 # NOTE: This dynamically typed stub was automatically generated by stubgen.
 
+from .base import Connection, Engine, RowProxy, Transaction
```

Comment:
interesting that `mypy` didn't complain. *shrug* forced an update with this feedback.